modules/op-release-notes-1-8.adoc

```diff
@@ -50,8 +238,141 @@ Workaround: Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image s
+// no link provided
+
+* With this update, `ClusterTask` resources no longer run as root by default, except for the `buildah` and `s2i` cluster tasks.
+// https://issues.redhat.com/browse/SRVKP-2385
+// Shubham Minglani
+
+* Before this update, tasks on {pipelines-title} 1.7.1 failed when using `init` as a first argument followed by two or more arguments. With this update, the flags are parsed correctly, and the task runs are successful.
+// https://issues.redhat.com/browse/SRVKP-2340 (borrowed from 1.7.2 release notes)
+// Vincent Demeester @vdemeester
+
+* Before this update, installation of the {pipelines-title} Operator on {product-title} 4.9 and 4.10 failed due to an invalid role binding, with the following error message:
++
+[source,terminal]
+----
+error updating rolebinding openshift-operators-prometheus-k8s-read-binding: RoleBinding.rbac.authorization.k8s.io "openshift-operators-prometheus-k8s-read-binding" is invalid: roleRef: Invalid value: rbac.RoleRef{APIGroup:"rbac.authorization.k8s.io", Kind:"Role", Name:"openshift-operator-read"}: cannot change roleRef
+----
++
+This update fixes the issue so that the failure no longer occurs.
+// (borrowed from 1.7.2 release notes) https://issues.redhat.com/browse/SRVKP-2327
+// Khurram Baig
+
+* Previously, upgrading the {pipelines-title} Operator caused `Pipeline`  objects to fail. This update fixes this issue. During upgrades, the Operator no longer recreates the `pipeline` service account. As a result, secrets attached to the `pipeline` service account persist over upgrades and the `Pipeline` objects continue working correctly.
+// https://issues.redhat.com/browse/SRVKP-2256
+// Kushagra Kulshreshtha
+
+* With this update, {pac} pods run on infrastructure nodes if infrastructure node settings are configured in the `TektonConfig` custom resource (CR).
+// no link provided
+
+* Previously, with the resource pruner, each namespace Operator created a command that ran in a separate container. This design consumed too many resources in clusters with a high number of namespaces. For example, to run a single command, a cluster with 1000 namespaces produced 1000 containers in a pod.
++
+This update fixes the issue. It passes the namespace-based configuration to the job so that all the commands run in one container in a loop. (link:https://issues.redhat.com/browse/SRVKP-2160[SRVKP-2160])
+// Pradeep Kumar
+
+* In {tekton-chains} you must define a secret called `signing-secrets` to hold the key used for signing tasks and images. However, before this update, updating the {pipelines-title} Operator reset or overwrote this secret and the key was lost. This update fixes the issue. Now, if the secret is configured after installing {tekton-chains} through the Operator, the secret persists, and it is not overwritten by upgrades.
+// https://issues.redhat.com/browse/SRVKP-2304
+// Piyush Garg
```

Comment:
@piyush-garg  Please review and approve. Thanks.

Reply:
looks good